templates/advantage/index.html

```diff
@@ -0,0 +1,240 @@
+            </section>
+          </li>
+        </ul>
+      </aside>
+    </div>
+  </div>
+
+  {{ accounts }}
+
+  <div class="row">
+    <div class="col-12">
+      <ul class="p-inline-list">
+        <li class="p-inline-list__item">Get instructions for using:</li>
+        {% if personal_account['affordances']['livepatch'] %}
+        <li class="p-inline-list__item">
+          <span class="p-contextual-menu">
+            <a href="#" class="p-contextual-menu__toggle" aria-controls="#menu-livepatch" aria-expanded="false" aria-haspopup="true">Livepatch</a>
+            <span class="p-contextual-menu__dropdown" id="menu-livepatch" aria-hidden="true" aria-label="submenu" style="min-width: 20rem; right: -9rem; padding: 1rem;">
+              <p>To activate Livepatch:</p>
+              <div class="p-code-copyable">
+                <input class="p-code-copyable__input" value="sudo ua enable livepatch" readonly="readonly">
+                <button class="p-code-copyable__action">Copy to clipboard</button>
+              </div>
+
+              <p>To deactivate Livepatch:</p>
+              <div class="p-code-copyable">
+                <input class="p-code-copyable__input" value="sudo ua disable livepatch" readonly="readonly">
+                <button class="p-code-copyable__action">Copy to clipboard</button>
+              </div>
+            </span>
+          </span>
+        </li>
+        {% endif %}
+        {% if personal_account['affordances']['esm'] %}
+        <li class="p-inline-list__item">
+          <span class="p-contextual-menu">
+            <a href="#" class="p-contextual-menu__toggle" aria-controls="#menu-esm" aria-expanded="false" aria-haspopup="true">ESM</a>
+            <span class="p-contextual-menu__dropdown" id="menu-esm" aria-hidden="true" aria-label="submenu" style="min-width: 20rem; right: -9rem; padding: 1rem;">
+              <p>To activate ESM:</p>
+              <div class="p-code-copyable">
+                <input class="p-code-copyable__input" value="sudo ua enable esm" readonly="readonly">
+                <button class="p-code-copyable__action">Copy to clipboard</button>
+              </div>
+
+              <p>To deactivate ESM:</p>
+              <div class="p-code-copyable">
+                <input class="p-code-copyable__input" value="sudo ua disable esm" readonly="readonly">
+                <button class="p-code-copyable__action">Copy to clipboard</button>
+              </div>
+            </span>
+          </span>
+        </li>
+        {% endif %}
+        {% if personal_account['affordances']['fips'] %}
+        <li class="p-inline-list__item">
+          <span class="p-contextual-menu">
+            <a href="#" class="p-contextual-menu__toggle" aria-controls="#menu-fips" aria-expanded="false" aria-haspopup="true">FIPS</a>
+            <span class="p-contextual-menu__dropdown" id="menu-fips" aria-hidden="true" aria-label="submenu" style="min-width: 20rem; right: -9rem; padding: 1rem;">
```

Comment:
This inline style is duplicated thrice indicating it should be abstracted to a class.